src/html/settings.html

```diff
@@ -314,13 +314,24 @@ <h2>Custom Integrations</h2>
       </div>
     </div>
     <div class="footer">
-      <div>
-        Version <span id="version"></span>
-      </div>
-      <div>
-        Changelog: <a href="http://toggl.github.io/toggl-button">http://toggl.github.io/toggl-button</a>
+      <div id="footer-content">
+        <div>
+          <div>Version <span id="version"></span></div>
+          <div>
+            Changelog: <a href="http://toggl.github.io/toggl-button">http://toggl.github.io/toggl-button</a>
+          </div>
+        </div>
       </div>
     </div>
+    <div id="review-prompt">
+      <span>Enjoying Toggl Button?</span>
+      <a target="_blank" rel="noopener">
```

Comment:
I believe this is the "correct" attributes for security 🔒 

```suggestion
      <a target="_blank" rel="noopener noreferrer">
```

Reply:
Done ✅